assets/package.json

```diff
@@ -25,7 +25,7 @@
     "watch:css": "nodemon -w src/styles -e scss -x \"npm run build:css\"",
     "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
     "watch:images": "nodemon -w src/img -x \"npm run build:images\"",
-    "watch:js": "rollup -c --watch"
+    "watch:js": "rollup -c --watch && nodemon -w node_modules/date-time-format-timezone/build/browserified/date-time-format-timezone-complete-min.js -x \"npm run build:scripts\" "
```

Comment:
This is not working. You should just copy the part in line 18 that you added to line 28.